### PR TITLE
Migrated to the new antd, fixed tests, fixed annotations uploading

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
One more try to implement #3623

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@dvkruchinin 

Hi, Dmitry. The test ``actions_objects/case_60_autoborder_feature.js`` fails time to time. ``AssertionError: Timed out retrying after 25000ms: Expected to find element: `.cvat-draw-rectangle-popover-visible`, but never found it.``


Could you please add it to your list of "unstable" tests?

**Comment by dvkruchinin** [comment]
> Could you please add it to your list of "unstable" tests?

@bsekachev Added. I'll see what's going on. But I suspect that the reason is deeper. This may not be a test problem, but the functions of interaction with Popover .

File: cvat-ui/src/components/cvat-app.tsx [hunk 1]
```diff
@@ -358,6 +359,8 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
                                     </Switch>
                                 </GlobalHotKeys>
                                 {/* eslint-disable-next-line */}
+                                <ExportDatasetModal />
```
Comment (ActiveChooN) [inline cvat-ui/src/components/cvat-app.tsx:362]: I saw it was related to the original PR, but could you explain, why do we need to move modal to cvat-app? As I see there is no other modals here

Comment (bsekachev) [reply cvat-ui/src/components/cvat-app.tsx:362]: As far as I know after dismounting it still stays in DOM, and current antd version adds some css IDs (based on fields names).
So, when we go between pages we get duplicated ids and corresponding errors. 

I've added ``destroyOnClose``, and than decided that this component is quite often (projects page, project page, task page, tasks page, annotation view) and put it here.

File: cvat-ui/webpack.config.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ module.exports = (env) => ({
         host: process.env.CVAT_UI_HOST || 'localhost',
         port: 3000,
         historyApiFallback: true,
+        host: process.env.CVAT_UI_HOST || 'localhost',
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat-ui/webpack.config.js:34]:
```suggestion
```

File: cvat-ui/src/components/cvat-app.tsx (hunk 1 above)
Comment (ActiveChooN) [reply cvat-ui/src/components/cvat-app.tsx:362]: I guess, better to keep it consistent and keep modals where they need. It only question of readability and I'm not against your decision

**Review by ActiveChooN — APPROVED** [review]

**Pull request merged**